plugins/ftdidmx/FtdiWidget.h

```diff
@@ -224,11 +224,15 @@ class FtdiInterface {
   /** @brief Toggle communications line BREAK condition on/off */
   bool SetBreak(bool on);
 
-  /** @brief Write data to a previously-opened line */
+  /** @brief Write data to a previously-opened line, DMX only */
```

Comment:
You could use Doxygen @pre to specify the line has already been opened.

Reply:
Not following you?